jupyter-js-widgets/src/widget_int.js

```diff
@@ -322,10 +353,10 @@ var IntSliderView = widget.DOMWidgetView.extend({
         var actual_value;
         if (this.model.get('_range')) {
             actual_value = ui.values.map(this._validate_slide_value);
-            this.readout.textContent = actual_value.join('-');
+            this.readout.textContent = this.valueToString(actual_value);
```

Comment:
It seems that the only time you call `valueToString` on a range is when you already know it is a range and you are already doing a map. Would it be simpler to just delete `valueToString` and do one more map here using the format function directly?